### PR TITLE
File Added :- Assignment 9

### PR DESCRIPTION
**Pull request by shubhamdalal49**

### CODE REVIEW

**Review by nileshp1122 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    box-sizing: border-box;
+    color: #f52f2f;
+    background-color: white;
+    padding: 7px 10px;
+    margin: 13px 4px;
+    font-weight: 600;
+}
+.first-background {
+    display: flex;
+    background-image: url("img/open-search-bg.png");
```
Comment (Pawan1005) [inline Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css:75]: image is not uploaded here you need to check that before pushing on git

File: Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+.upload-file {
+    margin-top: 9px;
+}
+.upload-icon {
+    width: 41px;
+    height: 41px;
+    float: left;
+    background-image: url("img/tj-sprite-icon.png");
```
Comment (Pawan1005) [inline Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css:135]: image is not uploaded here you need to check that before pushing on git

File: Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    box-shadow: 0.5px 0.87px 10px 0 rgb(0 0 0 / 7%);
+    background: #fff;
+}
+.top-ind-icon {
+    height: 28px;
+    width: 28px;
+    float: left;
+    margin-right: 15px;
+    background-repeat: no-repeat;
+    background-image: url("img/list.png");
```
Comment (Pawan1005) [inline Shubham_Dalal/HTML/ASSIGNMENT9/Shubham_assigment_9.css:213]: image is not uploaded here you need to check that before pushing on git

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
image is not uploaded here you need to check that before pushing on git

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**